tello_web_server.py

```
@@ -95,7 +95,12 @@ async def connect(self) -> Dict[str, Any]:
             # ソケット初期化
             self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
             self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-            self.socket.bind((self.local_ip, self.local_port))
+            try:
+                self.socket.bind((self.local_ip, self.local_port))
+            except OSError as e:
+                logger.error(f"ソケットバインドに失敗しました (port {self.local_port}): {e}")
+                self.socket.close()
+                raise ConnectionError(f"ポート {self.local_port} の使用に失敗しました: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)